include/grl/kuka/KukaJAVAdriver.hpp

```diff
@@ -288,9 +289,27 @@ namespace grl { namespace robot { namespace arm {
 
           auto name = fbbP->CreateString(std::get<RobotName>(params_));
 
-          auto kukaiiwaArmConfiguration = flatbuffer::CreateKUKAiiwaArmConfiguration(*fbbP,name,commandInterface_,monitorInterface_);
+          auto clientCommandMode = grl::flatbuffer::EClientCommandMode_POSITION;
```

Comment:
The udp packet that is sent across will contain grl::flatbuffer::EClientCommandMode_POSITION every time run_one is called (which should be as frequently as possible. This means I don't believe it will be possible to switch to any other command mode (wrench, torque). Perhaps that's all the currently supported functionality at the moment anyway...